core/tests/puppeteer-acceptance-tests/utilities/user/moderator.ts

```diff
@@ -0,0 +1,214 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview moderator user utility file.
+ */
+
+import {BaseUser} from '../common/puppeteer-utils';
+import testConstants from '../common/test-constants';
+import {showMessage} from '../common/show-message';
+
+export class Moderator extends BaseUser {
+  /**
+   * Function to navigate to the moderator page.
+   */
+  async navigateToModeratorPage(): Promise<void> {
+    await this.page.goto(testConstants.URLs.ModeratorPage, {waitUntil: 'load'});
```

Comment:
Also store the url as constant in this file, please.

Reply:
done.